Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Types/OperationMode.mo

```diff
@@ -0,0 +1,9 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.Types;
+type OperationMode = enumeration(
+    occupied "Occupied Mode",
+    coolDown "Cool-down Mode",
+    setup "Setup Mode",
+    warmup "Warmup Mode",
```

Comment:
If we use coolDown, we should also use warmUp rather than warmup to be consistent.